src/Actions/HandlesFailedJobsInterface.php

```diff
@@ -16,7 +16,7 @@
 use Sassnowski\Venture\WorkflowStepInterface;
 use Throwable;
 
-interface HandlesFailedJobs
+interface HandlesFailedJobsInterface
```

Comment:
For the actions I actually don't want to use the `Interface` suffix. I treat them like capabilities so the interface is something that `HandlesFailedJobs`.